README.md

```diff
@@ -7,13 +7,22 @@ You can use basic [glob syntax](https://en.wikipedia.org/wiki/Glob_(programming)
 
 ## How to install
 
-Simply drop the jar in `$KEYCLOAK_HOME\standalone\deployments`, it will be automatically deployed by keycloak.
+Simply drop the `jar` in the plugin directory, it will be automatically deployed by keycloak.
+
+### Keycloak < 17
+
+The plugin directory is `$KEYCLOAK_HOME\standalone\deployments`.
+
+### Keycloak >= 17
+
+The plugin directory is `$KEYCLOAK_HOME\providers`.
```

Comment:
The new folder doesn't come from the version but from the distribution type (wildfly or quarkus). Quarkus is now the default since v17